packages/elements/src/react/common/form/index.tsx

```diff
@@ -337,8 +337,13 @@ const useInput = ({
       type: 'text',
       spellCheck: false,
     };
-  }
-  if (type === 'password' && shouldValidatePassword) {
+  } else if (type === 'backup_code') {
```

Comment:
The `code` name has a number of predefined assumptions which don't work for backup codes. 

The `backup_code` name circumvents those assumptions while still maintaining the required field name of `code`.